docker-compose.yaml

```diff
@@ -63,4 +103,9 @@ services:
     environment:
       - NOTEBOOK_PORT=4041
       - SPARK_MASTER_URL=spark://spark-master:7077
-      - SPARK_DRIVER_HOST=spark-notebook
\ No newline at end of file
+      - SPARK_DRIVER_HOST=spark-notebook
+      - MINIO_URL=http://minio:9000
+      - MINIO_ACCESS_KEY=minio
+      - MINIO_SECRET_KEY=minio123
+    volumes:
+      - ./cdm_shared_workspace:/cdm_shared_workspace
```

Comment:
I thought from slack we didn't want this mount?

Reply:
we don't want this for Rancher. But this is just for local development.

Reply:
ok, maybe to avoid confusion we should replicate the rancher config here

Reply:
unnecessary change but 👍 . Rancher settings can change.

Reply:
:+1: